src/flake8/options/aggregator.py

```diff
@@ -43,6 +45,16 @@ def aggregate_options(
             assert isinstance(dest_val, str)
             dest_name = dest_val
 
+        if config_name in ["ignore", "extend-ignore"]:
+            for error_code in value:
+                if not re.match("^[A-Z][0-9]{3}$", error_code):
```

Comment:
this should reuse the same pattern that's used for validating entrypoints, notably they can be any error prefix here

Reply:
I don't like repeating myself but that's not right -- it should reuse the exact same pattern that's used for validating entrypoints

there should be no commas or spaces when it checks it

Reply:
Ooh wait you're referring to `VALID_CODE = re.compile("^[A-Z]{1,3}[0-9]{0,3}$", re.ASCII)`. 
I missed that initially, sorry.